### PR TITLE
ADEN-8507 Add logic for below the fold slots injection

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/ad-engine/services/slot-injector.ts [hunk 1]
```diff
@@ -54,6 +61,10 @@ class SlotInjector {
 			return null;
 		}
 
+		if (dryRun) {
+			return true;
```
Comment (fraszczakszymon) [inline src/ad-engine/services/slot-injector.ts]: Method returns now null|boolean|DOMElement.

File: src/ad-engine/services/slot-injector.ts [hunk 2]
```diff
@@ -54,6 +61,10 @@ class SlotInjector {
 			return null;
 		}
 
+		if (dryRun) {
```
Comment (fraszczakszymon) [inline src/ad-engine/services/slot-injector.ts]: As far as I understand `dryRun` might be not needed (?)
https://github.com/Wikia/app/pull/16842/files#r267385572

Comment (dmnsobczak) [reply src/ad-engine/services/slot-injector.ts]: Right, reverted.

File: src/ad-engine/services/slot-injector.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/ad-engine/services/slot-injector.ts]: N/A

**Pull request merged**